src/vector/getconfig.js

```diff
@@ -32,7 +34,7 @@ export async function getVectorConfig(relativeLocation) {
     }
 }
 
-function getConfig(configJsonFilename) {
+export function getConfig(configJsonFilename) {
```

Comment:
couldn't spot any usage of this outside of this module?

Reply:
There isn't, but this method was previously duplicated in index.js and there it was exported, I consolidated the two

Reply:
Makes it a bit clearer it's only used internally when making changing in the future ...

Reply:
Sure, will do